Dockerfile

```diff
@@ -40,20 +40,20 @@ RUN go mod download
 ARG MAKE='make build'
 RUN ${MAKE}
 
-FROM alpine:3.14
```

Comment:
Please add the following prior to `COPY . .`  so that docker build has a cache layer for dependencies which make repeated docker build more efficient. 

```
COPY go.mod vendor* ./
RUN [ ! -d "vendor" ] && go mod download all || echo "skipping..."
```